documentation/assemblies/managing/assembly-resource-status-access.adoc

```diff
@@ -3,9 +3,14 @@
 // assembly-management-tasks.adoc
 
 [id="resource-status-access_{context}"]
```

Comment:
Should we change the id when we changed a bit the purpose of the assembly? Maybe we can also rename the file to better correspond with the content? It would make it easier to find.

Reply:
Yep. I changed